### DISCUSSION TITLE
How do I move a file to a desired folder in lazyvim

### DISCUSSION THREAD

**Discussion by ghost** [nvim-neo-tree/neo-tree.nvim#1332] [Q&A]
Using a neotree setup to manage files. I am new to using lazyvim. Please direct me to help. THANKS !

**Comment by cseickel** [comment]
The default bindings are `y` to "yank" (copy) or `x` to "cut", then go to the desired destination an `p` to "put"/"paste"

**Reply by ghost** [reply]
@cseickel Thanks but, i asked for a file to moved to a folder not the codebase to move.

**Reply by cseickel** [reply]
I don't understand what you mean by "codebase to move". My instructions where for moving a file from one folder to another folder. Is that not what you were asking about?

**Reply by pysan3** [reply]
I think he means to move a file to a destination that is not in cwd. Especially `~/.config/nvim/...`, while vim is running on a very different location.

**Reply by ghost** [reply]
So, you mean to yank the file itself, or code lines written in it. That's what I meant by codebase. Because so far I used yank and paste just to copy code lines. I don't know if it's used for yanking the file.😅

**Reply by pysan3** [reply]
Yes, in neo-tree, yank means to yank the file, and when you `p` (paste) it somewhere, neo-tree will move the yanked file to that location.

If you want to copy the filepath (literal `~/path/blah/blah`), see this issue and you can add different keybinds. https://github.com/nvim-neo-tree/neo-tree.nvim/issues/1163
